Dockerize e2e tests (#64)
open-voip-alliance/WebphoneLib: README.md

documentation from the `jsdoc` comments in the source code. See [this
link](https://typedoc.org/guides/doccomments/) for more information on which
`jsdoc` tags are supported.

## Run puppeteer tests

### Using docker

To be able to run the tests with docker you can add a .env file and use the `npm run docker-build` command followed by the `npm run docker-run` command to run the puppeteer tests.

Add a .env file with the following:

```javascript
USER_A = '<user-a>';
USER_B = '<user-b>';
PASSWORD_A = '<password-user-a>';
PASSWORD_B = '<password-user-b>';
NUMBER_A = '<number-user-a>';
NUMBER_B = '<number-user-b>';
```

> Or you could use the following command (if you don't want to use a .env file) after building the docker image (`npm run docker-build`):

Reviewer: ```suggestion
Or if you don't want to use a .env file you could pass your configuration to the docker container:
```
